pages/api/propublica/member.js

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect } from 'react';
-import { headers } from './headers';
+import headers from '../../headers';
```

Comment:
Can you move the headers file into the propublica folder, to be more consistent with how our local folders are currently set up

Reply:
Fixed it. Thanks Voniel